proto/babylon/finality/v1/finality.proto

```diff
@@ -8,106 +8,135 @@ import "gogoproto/gogo.proto";
+  // addr is the bytes of the address to receive commission from delegations.
+  bytes addr = 2;
+  // commission defines the commission rate of finality provider
+  string commission = 3 [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec"
+  ];
+  // total_bonded_sat is the total amount of bonded BTC stake (in Satoshi) of
+  // the finality provider
+  uint64 total_bonded_sat = 4;
+  // is_timestamped indicates whether the finality provider
+  // has timestamped public randomness committed
+  // if no, it should not be assigned voting power
+  bool is_timestamped = 5;
+  // is_jailed indicates whether the finality provider
+  // is jailed, if so, it should not be assigned voting power
+  bool is_jailed = 6;
+  // is_slashed indicates whether the finality provider
+  // is slashed, if so, it should not be assigned voting power
+  bool is_slashed = 7;
 }
 
 // IndexedBlock is the necessary metadata and finalization status of a block
 message IndexedBlock {
-    // height is the height of the block
-    uint64 height = 1;
-    // app_hash is the AppHash of the block
-    bytes app_hash = 2;
-    // finalized indicates whether the IndexedBlock is finalised by 2/3
-    // finality providers or not
-    bool finalized = 3;
+  // height is the height of the block
+  uint64 height = 1;
+  // app_hash is the AppHash of the block
+  bytes app_hash = 2;
+  // finalized indicates whether the IndexedBlock is finalised by 2/3
+  // finality providers or not
+  bool finalized = 3;
 }
 
 // PubRandCommit is a commitment to a series of public randomness
 // currently, the commitment is a root of a Merkle tree that includes
 // a series of public randomness
 message PubRandCommit {
-    // start_height is the height of the first commitment
-    uint64 start_height = 1;
-    // num_pub_rand is the number of committed public randomness
-    uint64 num_pub_rand = 2;
-    // commitment is the value of the commitment
-    // currently, it is the root of the merkle tree constructed by the public randomness
-    bytes commitment = 3;
-    // epoch_num defines the epoch number that the commit falls into
-    uint64 epoch_num = 4;
+  // start_height is the height of the first commitment
+  uint64 start_height = 1;
+  // num_pub_rand is the number of committed public randomness
+  uint64 num_pub_rand = 2;
+  // commitment is the value of the commitment
+  // currently, it is the root of the merkle tree constructed by the public
+  // randomness
+  bytes commitment = 3;
+  // epoch_num defines the epoch number that the commit falls into
+  uint64 epoch_num = 4;
+}
+
+// PubRandCommitIndexValue represents a list of sorted start heights for public
+// randomness commitments.
+message PubRandCommitIndexValue {
+  option (gogoproto.goproto_getters) = false;
+  // A list of start heights corresponding to committed public randomness
+  // ranges.
+  repeated uint64 heights = 1;
 }
```

Comment:
only change here is the `PubRandCommitIndexValue`

Reply:
yes, the other diff is just formatting